src/test/smoketest/plain_server.http

```diff
@@ -0,0 +1,223 @@
+### Create Single Patient
```

Comment:
This appears to be a file rename. Shouldn't this pull request just rename `plain_server.rest`?